src/skibidi_orm/migration_engine/migration_element.py

```diff
@@ -1,15 +1,19 @@
 from skibidi_orm.migration_engine.adapters.base_adapter import BaseAdapter
+from typing import Any
 
 
 # All classes need to inherit from this class if they participate in a migration
 class MigrationElement:
     adapter: BaseAdapter
+    operations: list[Any] = []
 
-    @classmethod
-    def migrate(cls):
+    #   @classmethod
+    def migrate(self):
         table = MigrationElement.__subclasses__()
+
         for cls in table:
             # Do migration action
             # print(cls.__name__, cls.__dict__)
             table_instance = cls()
             table_instance.adapter.execute_migration()
```

Comment:
just a thought - do we have to initalize an instance here? no way to do it via the class?